components/NavBar.js

```diff
@@ -214,7 +214,7 @@ const NavBar = () => {
   }
 
   useEffect(() => {
-    window.addEventListener('scroll', handleScroll);
+    window.addEventListener('scroll', handleScroll());
     window.addEventListener('resize', handleResize);
```

Comment:
would we need to add parentheses to handleResize as well for the event listener on line 218 to work?

Reply:
Hmmm it seems to work without it 🤔 

`handleScroll` is a little different in that it returns another function, whereas `handleResize` is the actual function itself, so maybe that has something to do with it? I'm not completely sure either way though lol

Reply:
oooh i didn't look into it, it makes sense if it works without it since you just said they're different. sounds g!